browser/ui/brave_actions/shields_action_view_controller.cc

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/brave_actions/shields_action_view_controller.h"
+
+#include "brave/browser/ui/brave_actions/brave_action_icon_with_badge_image_source.h"
+#include "chrome/browser/extensions/extension_action.h"
+#include "chrome/browser/themes/theme_properties.h"
+#include "chrome/browser/sessions/session_tab_helper.h"
+#include "components/vector_icons/vector_icons.h"
+#include "ui/base/theme_provider.h"
+#include "ui/gfx/canvas.h"
+#include "ui/gfx/image/canvas_image_source.h"
+#include "ui/gfx/image/image_skia.h"
+#include "ui/gfx/paint_vector_icon.h"
+#include "ui/gfx/scoped_canvas.h"
+
+void ShieldsActionViewController::HideActivePopup() {
```

Comment:
I tested a couple of variations of opening popups and popups from other extensions and in all cases it did the same thing as chrome or other extensions in the normal extension bar. So I thought this would be ok for now and save the complexity of passing the ToolbarActionBar through LocationBarView to BraveActionsContainer into ShieldsActionViewController. Any thoughts?

Reply:
Ah, then Ok!